modules/aruco/src/aruco.cpp

```diff
@@ -750,9 +847,9 @@ static Point3f _interpolate2Dline(const std::vector<cv::Point2f>& nContours){
  * @return Crossed Point
  */
 static Point2f _getCrossPoint(Point3f nLine1, Point3f nLine2){
-	Matx22f A(nLine1.x, nLine1.y, nLine2.x, nLine2.y);
-	Vec2f B(-nLine1.z, -nLine2.z);
-	return Vec2f(A.solve(B).val);
+    Matx22f A(nLine1.x, nLine1.y, nLine2.x, nLine2.y);
+    Vec2f B(-nLine1.z, -nLine2.z);
+    return Vec2f(A.solve(B).val);
```

Comment:
Please avoid such massive space-only changes.
It is really hard to see/track the significant changes of PRs.

Consider moving these "whitespace" changes to the separate first commit or separate PR.